astropy/units/format/console.py

```diff
@@ -25,20 +35,20 @@ class Console(base.Base):
       2.1798721*10^-18 m^2 kg / s^2
     """
 
-    _times = "*"
-    _line = "-"
-    _space = " "
+    _times: ClassVar[str] = "*"
```

Comment:
I think these end up being quite useful because it's important to know when you're modifying a class-level variable. Lots of unexpected behavior can happen when changing a class variable and you think it's an instance variable.

Reply:
OK, that makes sense - and indeed these singleton classes are a bit odd.